packages/zoe/src/issuerTable.js

```diff
@@ -40,6 +40,12 @@ const makeIssuerTable = () => {
     /** @type {WeakStore<Issuer<any>,Brand<any>>} */
     const issuerToBrand = makeWeakStore('issuer');
 
+    const registerIssuerRecord = issuerRecord => {
```

Comment:
`registerIssuerKit`

Reply:
Naming problem: ERTP has a `makeIssuerKit` which returns a record that also contains an ERTP mint. Therefore, an "issuer kit" has such a field. Zoe already has an "IssuerRecord" abstraction that is similar except it has no mint field.